### ISSUE TITLE
Problem with MO2 installer on Linux Mint Cinnamom

### ISSUE THREAD

**Issue by DottorPeste** [rockerbacon/modorganizer2-linux-installer#740]
Steam installed via Flatpak, tried both Proton 9 and Experimental, updated everything possible. When i run the installer sheet through the console i get this message every time, also checked all the other sheets to no avail... (ps: im new to linux, so plz be gentle and explain things so that i can understand)

NFO: all dependencies met
INFO: selected game 'fallout4'
INFO: steam not found in ''
INFO: found Steam in '/home/dottorpeste/.var/app/com.valvesoftware.Steam/.local/share/Steam'
INFO: game found in '/home/dottorpeste/.var/app/com.valvesoftware.Steam/.local/share/Steam'
Traceback (most recent call last):
  File "/app/bin/protontricks", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/cli/main.py", line 32, in cli
    main(args)
  File "/app/lib/python3.11/site-packages/protontricks/cli/util.py", line 159, in wrapper
    return cli_func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/cli/main.py", line 339, in main
    proton_app = find_proton_app(
                 ^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 848, in find_proton_app
    tool_app = find_steam_compat_tool_app(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 609, in find_steam_compat_tool_app
    appinfo_sections = [
                       ^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 609, in <listcomp>
    appinfo_sections = [
                       ^
  File "/app/lib/python3.11/site-packages/protontricks/steam.py", line 531, in iter_appinfo_sections
    raise SyntaxError(
SyntaxError: Invalid file magic number. The appinfo.vdf version might not be supported by the current version of Protontricks - please check for updates.
ERROR: Protontricks did not find a valid prefix directory.  Stdout was:\n

**Comment by DottorPeste** [comment]
Ps: Thx in advance to anyone taking the time to help :)

**Comment by Stevetopop20** [comment]
Hello. I'm also new to linux however I just fixed this problem myself,

the problem was that I needed a flatpak-beta for it to work, the command lines are

flatpak remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo

followed by

flatpak install flathub-beta com.github.Matoking.protontricks

then when I used 

./install.sh

it worked.

**Comment by DottorPeste** [comment]
Hi. Ty for the a sfera. I tried italiani and, when i booted the remote-add it didnt give me back any info, on the console. I then assumed it was all right, cuz no errors came up. But then, when i run install it says to me:
Error: Nothing matches com.github.Matoking.protontricks in remote flathub-beta
What now?

**Comment by Stevetopop20** [comment]
I don't fully know your setup however here are the links to how I got mods to work on skyrim SE and I can't imagine Fallout is too far from these steps.

The main guild I was following: https://www.youtube.com/watch?v=OEbZ3hNcoRg

Where I learned I need a flatpak-beta: https://github.com/Matoking/protontricks.

I don't remember if I used this website but it was in my history: https://discourse.flathub.org/t/how-to-use-flathub-beta/2111

I hope these resources aid you, for me its late and I must sleep.
